banzai_floyds/main.py

```diff
@@ -59,6 +60,26 @@ def populate_photometric_standards():
     banzai_floyds.dbs.ingest_standards(args.db_address)
 
 
+def add_order_location():
+    parser = argparse.ArgumentParser(inspect.getdoc(banzai_floyds.dbs.add_order_location))
+    parser.add_argument('--db-address', dest='db_address',
```

Comment:
Should we check for this in the environment?

Reply:
All of our other cli scripts don't so I'd rather be explicit.